hhub/views.py

```diff
@@ -60,10 +68,15 @@ def search_entries(request):
     typetag = request.GET.get('typetag', '')
     tags = request.GET.get('tags', '')
     platform = request.GET.get('platform', '')
+    sort_by = request.GET.get('sort_by', 1)
```

Comment:
Why is the default value 1 when everything else here is set to `''`?

Reply:
lemme guess: same goes for group_by?